app/models/mdm/web_page.rb

```diff
@@ -1,4 +1,5 @@
 # Web page requested from a {#web_site}.
+require 'webrick'
```

Comment:
Ah, that happens because the encoded YAML has a class reference that it tries to reify to a real class, but it's not loaded without the `require`.  This makes WEBrick a dependency of the gem as a whole and so the require should move to Core requires [here](https://github.com/rapid7/metasploit_data_models/blob/master/lib/metasploit_data_models.rb#L1-L4)